go.mod

```diff
@@ -63,6 +63,131 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require (
```

Comment:
Why do you need this separate require block?

Reply:
Not sure why go mod tidy created a new block, I will merge them.

Reply:
Go 1.17 release notes mention this explicitly
> Because the number of explicit requirements may be substantially larger in an expanded Go 1.17 go.mod file, the newly-added requirements on indirect dependencies in a go 1.17 module are maintained in a separate require block from the block containing direct dependencies. 

I think it should be kept separate (it helps distinguish direct and indirect dependencies)

Reply:
Thanks @mx-psi !